pynn_spinnaker/random.py

```diff
@@ -38,6 +39,11 @@ def __init__(self, host_rng, seed=None):
         assert host_rng is not None
         self._host_rng = host_rng
 
+        # Generate four word base seed for SpiNNaker RNGs
+        self.SeedWords = 4
```

Comment:
This doesn't need to be initialised on each instance - just stick it up with the _dist_XXXXX stuff (probably also _scored as it's not a PyNN standard thing